Logger/LogFactory.cs

```diff
@@ -2,10 +2,32 @@
 {
     public class LogFactory
     {
-        public BaseLogger CreateLogger(string className)
-        {
+        public BaseLogger? CreateLogger()
+        { 
+            if (FilePath == null)
+            {
+                {
+                   
+                };
+                return null;
+            }
+            else
+            {
+                BaseLogger baseLogger = new FileLogger(FilePath)
+                {
+                    ClassName = nameof(FileLogger),
```

Comment:
```suggestion
                    ClassName = className,
```
`className` should be an input parameter on this method of type string.